### ISSUE TITLE
New directory listing not working for new tiles locations

### ISSUE THREAD

**Issue by zstadler** [IsraelHikingMap/Site#286]
Following issue IsraelHikingMap/Map#8, maps tiles are now available at `/<Language>/<Tile set>` where `<Language` is either `Hebrew` or `English` and `<Tile set>` is either `Tiles`, `mtbTiles`, or `OverlayTiles`.

New virtual directories were configured and `web.config` was updated on the server, the tiles are accessible, but the new directory listing is not working for the new locations. 

For example: http://israelhiking.osm.org.il/Hebrew/Tiles/

**Comment by HarelM** [comment]
Actually it is working according to what you defined in the ```web.config``` file:
http://israelhiking.osm.org.il/Hebrew_Tiles/
The problem here is that there's no physical ```Hebrew``` folder, if there was we could have define the listing to use this folder and see the relevant sub folders tile sets.
Can you remind me why that wan't the final implementation?

**Comment by zstadler** [comment]
Tiles were never a physical directory. They are created within the directory structure of the Map repository installation.

**Comment by HarelM** [comment]
Fixed the configuration file, ignore my previous comment. 
Please have a look now and let me know if this is fine by you.
http://israelhiking.osm.org.il/Hebrew/Tiles/
This also works:
http://israelhiking.osm.org.il/Hebrew/
http://israelhiking.osm.org.il/English/

**Comment by HarelM** [comment]
No, Sorry, The English doesn't work, can you explain what procedure you are doing to enable regular listing?

**Comment by HarelM** [comment]
I have added permission for: IIS AppPool\\<```site name```> in IIS so that the English folder will be visible from the site's user. I think it will take a while to change all the permissions in all the files... I'll check it in the morning.

**Comment by zstadler** [comment]
It's a workaround. There was no intention to allow directory listing for the Hebrew, English or Hebrew/Oruxmaps directories.

Directory listing is needed only for the specific tiles directories under the language directory.

**Comment by HarelM** [comment]
I don't see how this is a workaround - sounds like this is what you wanted to accomplish.
If I understand you correctly you want the address to be ```...IHM.../Hebrew/Tiles``` but you don't want the following address to work ```...IHM.../Hebrew/```? doesn't sound correct to me...

**Comment by zstadler** [comment]
You got it right. 

Before adding English tiles, the parent directory of the Hebrew tiles did not have directory listing enabled. I want to keep it this way as `Hebrew` had, and probably will have in the future, contents that should not be listed.

**Comment by HarelM** [comment]
Sorry, I respectfully do not agree. as I said, it doesn't make sense address wise. the tiles address are (and I think it is correct this way) hebrew/tiles/... - surfing to /hebrew/ will return an error and surfing to /hebrew/tiles won't - doesn't sound correct to me.
The old format is not comparable since the root was each tiles folder so the parent was the regular map site.
I think it will easier and more intuitive to keep the same structure in the file system as well.
I'm sure you can keep these folders clean in term of unwanted files as you do with the English folder.
As an example the fact that I can see the OruxMap folder inside there is very good IMO since I know which language has an offline map and which doesn't.

Bottom line, I don't think I should write a complex mechanism just to support this, it doesn't make sense to me.

**Comment by zstadler** [comment]
I accept.

**Comment by HarelM** [comment]
Yay!
